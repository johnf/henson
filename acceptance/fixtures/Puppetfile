forge "https://forge.puppetlabs.com/"

# puppet forge
mod "puppetlabs/cloud_provisioner", "1.0.5"

# path
mod "ssh",       :path => "#{File.expand_path("../modules/ssh", __FILE__)}"

# git
mod "stdlib",    :git => "git://github.com/puppetlabs/puppetlabs-stdlib", :tag => "3.0.0"
mod "openstack", :git => "git://github.com/puppetlabs/puppetlabs-openstack", :branch => "folsom"
mod "lvm",       :git => "git://github.com/puppetlabs/puppetlabs-lvm", :ref => "8a7b20e3"

# github release/tag
mod "boxen", "~> 0.1.0", :github => "boxen/puppet-boxen"
