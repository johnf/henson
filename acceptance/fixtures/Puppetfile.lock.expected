FORGE
  remote: https://forge.puppetlabs.com/
  specs:
    cloud_provisioner (1.0.5)

GIT
  remote: git://github.com/puppetlabs/puppetlabs-lvm
  specs:
    lvm (8a7b20e3)

GIT
  remote: git://github.com/puppetlabs/puppetlabs-openstack
  specs:
    openstack (somesha)

GIT
  remote: git://github.com/puppetlabs/puppetlabs-stdlib
  specs:
    stdlib (somesha)

GITHUB
  remote: https://api.github.com/
  specs:
    boxen (0.1.9)

PATH
  remote: "."
  specs:
    ssh (0.0.0)
